### PR TITLE
[DOCS] Mention mmap and FD limits when increasing default max shard per node

### PR DESCRIPTION
**Pull request by pxsalehi**

### CODE REVIEW

File: docs/reference/setup/sysconfig/virtual-memory.asciidoc [hunk 1]
```diff
@@ -18,3 +18,10 @@ To set this value permanently, update the `vm.max_map_count` setting in
 
 The RPM and Debian packages will configure this setting automatically. No
 further configuration is required.
+
+You can find out the current mmap count of a running Elasticsearch process using the following command:
```
Comment (DaveCTurner) [inline docs/reference/setup/sysconfig/virtual-memory.asciidoc:22]: I reckon it's more usable not to split this sentence either side of the displayed example:

```suggestion
You can find out the current mmap count of a running Elasticsearch process
using the following command, where `$PID` is the process ID of the running
Elasticsearch process:
```

File: docs/reference/setup/sysconfig/virtual-memory.asciidoc [hunk 2]
```diff
@@ -18,3 +18,10 @@ To set this value permanently, update the `vm.max_map_count` setting in
...
+[source,sh]
+----------------------------
+ls /proc/$PID/maps | wc -l
```
Comment (DaveCTurner) [inline docs/reference/setup/sysconfig/virtual-memory.asciidoc:25]: Sorry I gave you duff info here, `/proc/$PID/maps` is just a file:

```suggestion
wc -l /proc/$PID/maps
```

File: docs/reference/setup/sysconfig/virtual-memory.asciidoc [hunk 3]
```diff
@@ -18,3 +18,10 @@ To set this value permanently, update the `vm.max_map_count` setting in
...
+----------------------------
+Where `$PID` is the process ID of the running Elasticsearch process.
```
Comment (DaveCTurner) [inline docs/reference/setup/sysconfig/virtual-memory.asciidoc:27]: ... and therefore drop this

```suggestion
```

**Review by DaveCTurner — COMMENTED** [review]
Couple of nits but looks good.

File: docs/reference/setup/sysconfig/virtual-memory.asciidoc (hunk 2 above)
Comment (pxsalehi) [reply docs/reference/setup/sysconfig/virtual-memory.asciidoc:25]: I would have tried it first. But that doesn't work on Mac!

**Review by DaveCTurner — APPROVED** [review]
LGTM

**Pull request merged**